tests/integration/records/timezone.py

```diff
@@ -13,5 +15,5 @@ def set_session_tz(conn):
     # being of the session timezone.
     if conn.engine.name == 'vertica':
         tz = DB_SESSION_TIMEZONE
-        conn.execute('SET TIME ZONE TO :tz', tz=tz)
+        conn.execute(text('SET TIME ZONE TO :tz'), {'tz': tz})
```

Comment:
yeahhhh this is how you should do the format strings!

Reply:
So this works for real parameters, but doesn't work for table or schema names (probably on purpose on SQLAlchemy's part, they want you to use the much safer `Table` object.